app/views/_pod-template.html

```diff
@@ -7,188 +7,43 @@
   addHealthCheckUrl (optional)
  -->
 
- <!-- Prompt to add health checks if missing. -->
+<!-- Prompt to add health checks if missing. -->
 <div ng-if="detailed && addHealthCheckUrl && !(podTemplate | hasHealthChecks)" class="alert alert-info">
   <span class="pficon pficon-info" aria-hidden="true"></span>
-  <span ng-if="podTemplate.spec.containers.length === 1">This container has no health checks</span>
-  <span ng-if="podTemplate.spec.containers.length > 1">Not all containers have health checks</span>
+  <span ng-if="podTemplate.spec.initContainers.length === 1">This container has no health checks</span>
+  <span ng-if="podTemplate.spec.initContainers.length > 1">Not all containers have health checks</span>
```

Comment:
Init containers do not have health checks. We should still check `spec.containers.length` here.

Maybe change "This container" to "Container <name> does not..." if there is only one container. Then it's clear "This container" does not refer to the init container.